sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/StitchedColumn.java

```diff
@@ -12,16 +12,19 @@
 public class StitchedColumn extends AbstractColumnImpl<UUID> {
     private Column<DeltaKey> _oldColumn;
     private ByteBuffer _content;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_content" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00116)